### ISSUE TITLE
question: is PEP 701 going to be a problem for Spyder ? 

### ISSUE THREAD

**Issue by stonebig** [spyder-ide/spyder#20270]
I'm thinking about that specific one 

https://discuss.python.org/t/pep-701-syntactic-formalization-of-f-strings/22046/24

![image](https://user-images.githubusercontent.com/4312421/209827654-50680c1c-a46e-47b2-bdc2-02adc30404ba.png)

**Comment by stonebig** [comment]
maybe not, if jedi is ok  with it ? https://github.com/davidhalter/jedi/issues/1901

**Comment by dalthviz** [comment]
Hi @stonebig , thanks for reaching out and letting us know about this! I'm not sure but if jedi supports is it should be okay (at least in terms of completion functionality and code introspection). Maybe could be some issues with the syntax highlighting? 

Just in case, checking how currently writting something like that looks like in Spyder:

![imagen](https://user-images.githubusercontent.com/16781833/209854636-c679d4db-a07d-420f-a2ad-a4d6c9b887fb.png)

And just checking, this is how the completion works (puts single quotes for the dict key name):

![dict](https://user-images.githubusercontent.com/16781833/209854955-1708f381-9e1b-4f2c-bd3b-a3ddd2d3c2cb.gif)


What do you think @ccordoba12 ?

**Comment by ccordoba12** [comment]
I think we need to deal with switch to a new syntax highlighter to be able to deal with this use case (which seems hard to parse).
